bokeh/core/_templates/autoload_js.js

```diff
@@ -74,12 +74,10 @@ calls it with the rendered model.
       document.getElementsByTagName("head")[0].appendChild(s);
     }
   };
-
   {%- if elementid -%}
   var element = document.getElementById({{ elementid|json }});
   if (element == null) {
     console.log("Bokeh: ERROR: autoload.js configured with elementid '{{ elementid }}' but no matching script tag was found. ")
-    return false;
```

Comment:
The issue is that we now optionally publish an html div for the banner, then publish the autoload script. When the banner isn't loaded, this elementid doesn't exist and the load script is returned before embedding the resources.

I need to think a little more about the impact of this change on other autoload situations.